README.md

```diff
@@ -2,12 +2,77 @@
 
 This is the Android app for the Carleton University Science Student Success Centre. You can find the SSSC Server code [here](https://github.com/ScienceStudentSuccessCentre/SSSC-Server).
 
+- [Project summary](#project-summary)
+- [Updates](#updates)
+- [Getting started](#getting-started)
+  - [Downloading the project](#downloading-the-project)
+  - [Setting up an emulator](#setting-up-an-emulator)
+- [Uploading to the Play Store](#uploading-to-the-play-store)
+  - [Part 1: Generating a Signed APK](#generating-a-signed-apk)
+  - [Part 2: Creating a Release in the Play Console](#part-2-creating-a-release-in-the-play-console)
+  - [Part 3: Filling in app details on App Store Connect](#part-3-filling-in-app-details-on-app-store-connect)
+
 ## Project summary
 
 This project is the mirror version of the iOS Science Student Success Centre app, for Android! It allows students to view all of the events that are shown on the SSSC's website and quickly access the SSSC's resources page. 
 Event data for both the Android and iOS apps are retrieved using a GET request to the SSSC Server (linked above) using the `/events` endpoint, to ensure their data is sychronized. 
 
+## Updates
+
+(Refer to the ios app for the following features)
```

Comment:
Can you link the iOS app repo here using a markdown link? [Like this](https://github.com/ScienceStudentSuccessCentre/SSSC-iOS-App). (Also maybe proper capitalization of iOS!)